engine/configs/.env.example

```
@@ -43,6 +43,24 @@ MISTRAL_API_KEY=
+# SLUICE GATEWAY (Optional)
+# ===========================================
+# phoenixvc's OpenAI-compatible LiteLLM gateway. Routing through it centralises
+# spend tracking and per-key budgets instead of using vendor keys directly.
+# Both variables are required together — either alone leaves Sluice disabled.
+#
+# Requests are tagged with metadata.app / metadata.agent, which Sluice ADR 17
+# makes mandatory for first-party services. codeflow-engine's virtual key is
+# classified as a service, so untagged requests will be rejected with HTTP 400
+# once enforcement is enabled. See codeflow_engine/core/llm/sluice.py.
+SLUICE_BASE_URL=
+SLUICE_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SLUICE_API_KEY key should go before the SLUICE_BASE_URL key

(UnorderedKey)